dataplane/linux/vxlan_mgr.go

```diff
@@ -221,25 +315,38 @@ func (m *vxlanManager) KeepVXLANDeviceInSync(mtu int) {
 			time.Sleep(1 * time.Second)
 			continue
 		}
+
+		if parent, err := m.getLocalVTEPParent(); err != nil {
+			logrus.WithError(err).Warn("Failed configure VXLAN tunnel device, retrying...")
+			time.Sleep(1 * time.Second)
+			continue
+		} else {
+			if m.parentRouteTable == nil {
```

Comment:
I think we're working with this parentRouteTable field across multiple goroutines which means we should probably add some mutexes for safety.

An easy pattern for this looks like:

```
func getRouteTable() {
  m.Lock()
  defer m.Unlock()
  return m.parentRouteTable
}
```

Reply:
right, I'll update that